packages/fxa-react/styles/links.css

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.link-blue {
+  @apply underline text-blue-500 rounded-sm;
+
+  &:active {
+    @apply text-blue-800;
+  }
+}
+
+.link-grey {
+  @apply underline text-grey-500 rounded-sm;
+
+  &:active {
+    @apply text-grey-500;
+  }
+}
+
+@media (hover: hover) {
+  .link-blue:hover {
+    @apply text-blue-600;
+  }
+  .link-grey:hover {
+    @apply text-grey-500;
```

Comment:
Just to confirm, the ".link-gray:hover" should have the same text color for the "normal", ":active", _and_ ":hover" states?
The `.link-blue` has a normal color of "text-blue-500", an ":active" color of "text-blue-800", and a ":hover" color of "text-blue-600".

Reply:
Huh yeah, that doesn't seem right. `grey-600` is _super_ dark, I'm not sure why we didn't just call it `grey-900` to begin with, so changing the active and hover states to 600 looks like too big of a color difference to me. We might need a new grey, so I'll make note of this to ask in a design review (or just file a new ticket for it).